### ISSUE TITLE
Add unique value count to analysis

### ISSUE THREAD

**Issue by connerxyz** [variety/variety#110]
I find acquiring the number of unique values per field is a common general use-case that goes right alongside this type of data survey tool.  An option for this feature would be really great.

**Comment by todvora** [comment]
Hi @connercowling, do you mean something like the traditional ```SELECT COUNT(DISTINCT(my_field)```? 

Does the following command return what you expect? 
```js
db.my_collection_name.aggregate([
  {$group:{_id:"$my_field_name"}}, 
  {$group:{_id:'my_field_name',count:{$sum:1}}}
]);
```


Thanks,
Tomas

**Comment by connercowling** [comment]
Exactly.  It would be great to pass a variable and get/persist these results.  I find this is very handy for spotting fields with all the same value, fishy values for what you expect to be boolean, comparing datasets generated by same process but from different sources, etc.

**Comment by connercowling** [comment]
I'll send a pull-request if I can find the time to implement.

**Comment by todvora** [comment]
Sounds good, let me know, if you need any help with it.

**Comment by JamesCropcho** [comment]
Hello all, I'm closing this ticket for now since it's been inactive for two weeks. But by all means, feel free to re-open and work on this feature further, if you'd like and have the time.

All the Best,
James
